concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-47	47	to forgive	1528	FORGIVE
+RonaTas-2011-431-48	48	buckler with handle, handle of different arms		
+RonaTas-2011-431-49	49	knot, bend	760	KNOT
+RonaTas-2011-431-50	50	stack of hay		
+RonaTas-2011-431-51	51	tassel, fringe, tuft, node	2989	FRINGE
+RonaTas-2011-431-52	52	burdock, Arctium sp.		
+RonaTas-2011-431-53	53	ankle	803	ANKLE
+RonaTas-2011-431-54	54	ankle boot		
+RonaTas-2011-431-55	55	anthill	1873	ANTHILL
+RonaTas-2011-431-56	56	wine	1524	WINE
+RonaTas-2011-431-57	57	to cover, to overturn	761	COVER
+RonaTas-2011-431-58	58	cover	761	COVER
+RonaTas-2011-431-59	59	calf	1989	CALF
+RonaTas-2011-431-60	60	pepper	1385	PEPPER
+RonaTas-2011-431-61	61	pea, Pisum sativum; bean	832	BEAN
+RonaTas-2011-431-62	62	badger, Meles meles	3717	BADGER
+RonaTas-2011-431-63	63	witch, sorceress	824	WITCH
+RonaTas-2011-431-64	64	to annoy	2892	MAKE ANGRY (ANNOY)
+RonaTas-2011-431-65	65	revenge	495	REVENGE
+RonaTas-2011-431-66	66	stick, cane	1295	STICK
+RonaTas-2011-431-67	67	a kind of beer, alcoholic beverage	404	ALCOHOL (FERMENTED DRINK)
+RonaTas-2011-431-68	68	rich; abundant, roomy	712	RICH
+RonaTas-2011-431-69	69	horsefly, gadfly	3262	HORSEFLY
+RonaTas-2011-431-70	70	fast(ing)	1631	FAST
+RonaTas-2011-431-71	71	wise	698	WISE
+RonaTas-2011-431-72	72	cradle	2946	CRADLE
+RonaTas-2011-431-73	73	bison	3142	BISON
+RonaTas-2011-431-74	74	prison, jail	1281	PRISON
+RonaTas-2011-431-75	75	Ismaelite, name of an ethnic group in Medieval Hungary		
+RonaTas-2011-431-76	76	sad, bad-tempered	699	SAD
+RonaTas-2011-431-77	77	farewell, indulgence, dedication festival, annual parish-feast	3554	GOODBYE
+RonaTas-2011-431-78	78	ox; also ox, goat, ram without or with small or unusual horns;stupid	1502	GOAT
+RonaTas-2011-431-79	79	Volga Bulgar		
+RonaTas-2011-431-80	80	steam, vapour, fumes; heat	1193	STEAM
+RonaTas-2011-431-81	81	wheat, Triticum	1077	WHEAT
+RonaTas-2011-431-82	82	club, mace, reed-mace	1763	CLUB
+RonaTas-2011-431-83	83	beech	1950	BEECH
+RonaTas-2011-431-84	84	sin	2971	SIN
+RonaTas-2011-431-85	85	(blunt) end of something, a rod with an end cut, obtuse end of a field-strip, the shorter side or end of a piece of furniture, foot of the hill		
+RonaTas-2011-431-86	86	magical		
+RonaTas-2011-431-87	87	a kind of willow, Salix elægnos		
+RonaTas-2011-431-88	88	trap; snare (for catching birds); bird cage	1075	TRAP (PITFALL)
+RonaTas-2011-431-89	89	a kind of willow, used for baskets and in viticulture, Salix purpurea		
+RonaTas-2011-431-90	90	a kind of willow		
+RonaTas-2011-431-91	91	a kind of wolfs milk, Easter flower, Euphorbia		
+RonaTas-2011-431-92	92	a fish, Leuciscus souffia agassizi	227	FISH
+RonaTas-2011-431-93	93	only, just	1580	ONLY
+RonaTas-2011-431-94	94	pick-axe; mace; a small axe; war hammer		
+RonaTas-2011-431-95	95	nettle, Urtica	33	NETTLE
+RonaTas-2011-431-96	96	thicket, brushwood, dry foliage, grove, meadow	2819	GRASSLAND
+RonaTas-2011-431-97	97	thicket, brushwood; dry foliage, grove, meadow	2819	GRASSLAND
```

Comment:
This seems like a duplicate entry and should be removed

Reply:
If it is in the original list, I would leave it as is. @martino-vic Could you double-check with the original list?

Reply:
Just checked in the original source and it's treated as two separate cognate sets because of a possible metathesis. So one is `t͡ʃ ɒ l i t <  t͡ʃ a l t ɯ`, the other `t͡ʃ ɒ l i t < t͡ʃ a l ɯ t`. See https://github.com/martino-vic/rtbwestoldturkic/blob/main/cldf/forms.csv#L376

Reply:
Partially fixed this in 0ca3d47d736423dc3eb770614d9a9b3f5a89c47e
Since libre office, excel and google sheets all do unexpected formating (quotations marks, turning numbers into dates, autranslating words like "FALSE" to "FALSCH"), I have to use some other tool to re-numerate this column. Will try with pandas once I'm through editing all issues

Reply:
If the original list contains this line, I would leave it in. We want to reflect the original list as accurately as possible, and someone may want to look up the two cognate sets at some point.